src/riak_exporter/exporter.py

```diff
@@ -70,6 +70,28 @@ def parse_riak_stats_data(self, riak_stats_data):
                 prom_str = "riak_{} {}".format(key, prom_value)
                 yield prom_str
 
+    def parse_riak_repl_stats_data(self, riak_repl_stats_data):
+        """
+        Filter key/values pairs from collected Riak repl stats, which may be meaningful for Prometheus.
+
+        :param riak_repl_stats_data: Collected Riak repl stats object.
+        :type riak_repl_stats_data: dict
+        :return: Generator of metrics that can be put into Prometheus snapshot.
+        """
+        for key, value in riak_repl_stats_data.items():
+            if isinstance(value, (int, float, bool)):
+                prom_value = float(value)
```

Comment:
Will this not parse `True` as 1.0? As long as we know that it's fine, but it could be confusing depending on the key name.

Reply:
I kind of just did the same as the current parsing logic. 
That being said, I guess that parsing `True` as `1.0` is fine as this data would be shown in `grafana`